torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -475,9 +468,14 @@ void ProfilingGraphExecutorImpl::runNoGradOptimizations(
     for (const auto& passPair : getCustomPostPasses()) {
       passPair.first(graph);
     }
+    GRAPH_DEBUG("After customPostPasses \n", *graph);
+    if (tensorExprFuserEnabled()) {
```

Comment:
We want to call `RemoveTensorTypeSpecializations` regardless of whether or not `TE` is enabled. Let's make it the contract that the individaul `customPostPasses` don't need to invoke it themselves, that way they will compose. 

Should we also remove it here ? cc @jjsjann123  https://github.com/pytorch/pytorch/blob/0dc3f829d9d11ca7e972cb5fcc95059742fa71ea/torch/csrc/jit/codegen/cuda/graph_fuser.cpp#L2042

Reply:
Yes we can also remove the pass used in nvfuser.